docs/services/slam/cartographer/_index.md

```diff
@@ -101,70 +90,17 @@ If you choose to use movement sensor data for SLAM, you can:
 ### Create a new map
```

Comment:
Could fix while we're here on L88 above: The "If you choose this option..." bullet probably makes sense to be a sub-bullet of the "Add both IMU and odometry data" bullet